Jenkinsfile

```diff
@@ -0,0 +1,107 @@
+def buildScmInfo
+stage("Build") {
+    node('test-dynamic-slave') {
+        cleanWs()
+
+        sfScmInfo = checkout([$class: 'GitSCM',
+            branches: [[name: '*']],
+            doGenerateSubmoduleConfigurations: false,
+            extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'snowflake']],
+            submoduleCfg: [],
+            userRemoteConfigs: [[credentialsId: 'a0395839-84c7-4ceb-90e2-bcf66b2d6885', url: 'ssh://bitbucket-internal.int.snowflakecomputing.com:7999/opfdb/fdb_snowflake.git']]
```

Comment:
Is this sensitive information? Maybe you want to hide it.

Reply:
The file shouldn't be in there, I'll make a rebase to remove it from the history. But this isn't sensitive.